cogs/tiles.py

```diff
@@ -34,17 +36,22 @@ class Tiles(Cog):
             if query.lower() == key.lower():
                 obj = val
                 break
-        # search for partial matches if necessary
         if obj is None:
             if len(query) < 3:
                 msg = "Sorry, that specific object wasn't found, and it's too short to search."
                 return await ctx.send(msg)
-            for key, val in qindex.items():
-                if query.lower() in key.lower():
-                    obj = val
-                    break
         if obj is None:
```

Comment:
Good catch, the second `if` was redundant. Since there is a `return` in the code block of the first `if`, everything after that is an implicit `else`, so just removing the second `if` saves a level of indentation.